mappings/net/minecraft/fluid/Fluid.mapping

```diff
@@ -27,10 +27,10 @@ CLASS net/minecraft/class_3611 net/minecraft/fluid/Fluid
 		ARG 1 fluid
 	METHOD method_15781 setDefaultState (Lnet/minecraft/class_3610;)V
 		ARG 1 state
-	METHOD method_15782 getVelocity (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;Lnet/minecraft/class_3610;)Lnet/minecraft/class_243;
+	METHOD method_15782 getFlowDirectionVector (Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;Lnet/minecraft/class_3610;)Lnet/minecraft/class_243;
 		ARG 1 world
 		ARG 2 pos
-		ARG 3 state
+		ARG 3 mainFluidState
```

Comment:
I kinda prefer mainFluidState because it tells me that from all the other states in the method, that is the main one related to the desired operation. But since in most of the code the name is just `state` I guess it makes sense to keep it the same. I have changed it back in https://github.com/FabricMC/yarn/pull/3284/commits/5076644a8c3933dec14c80c9f6b0da20b352fffc